packages/grafana-data/src/utils/i18n.ts

```diff
@@ -0,0 +1,65 @@
+const ENGLISH_US = 'en-US';
+const FRENCH_FRANCE = 'fr-FR';
+const SPANISH_SPAIN = 'es-ES';
+const GERMAN_GERMANY = 'de-DE';
+const BRAZILIAN_PORTUGUESE = 'pt-BR';
+const CHINESE_SIMPLIFIED = 'zh-Hans';
+const ITALIAN_ITALY = 'it-IT';
+const JAPANESE_JAPAN = 'ja-JP';
+const INDONESIAN_INDONESIA = 'id-ID';
+const KOREAN_KOREA = 'ko-KR';
+const RUSSIAN_RUSSIA = 'ru-RU';
+const CZECH_CZECHIA = 'cs-CZ';
+const DUTCH_NETHERLANDS = 'nl-NL';
+const HUNGARIAN_HUNGARY = 'hu-HU';
+const PORTUGUESE_PORTUGAL = 'pt-PT';
+const POLISH_POLAND = 'pl-PL';
+const SWEDISH_SWEDEN = 'sv-SE';
+const TURKISH_TURKEY = 'tr-TR';
+const CHINESE_TRADITIONAL = 'zh-Hant';
+
+/**
+ * Pseudo locale.
+ */
+export const PSEUDO_LOCALE = 'pseudo';
```

Comment:
think pseudo should probably stay within core for now? i'm not sure what plugins would do with it

Reply:
Fixed 👍